### DISCUSSION TITLE
Does Nova prune older notifications?

### DISCUSSION THREAD

**Discussion by ziming** [laravel/nova-issues#4238] [Q&A]
More of a question.

Does Nova prune old notifications after a certain period of time or after a certain number (say 50) for a user?

Or is this something where we need to write our own scheduled command to do it?

What happens when the notifications count get huge, do we get a link to view all notifications on a seperate page or the notifications sidebar will just get very long to scroll?

**Comment by RhysLees** [comment]
I think we should add a `nova:prune-notifications` command like Laravel does with a lot of other stuff
